modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/CustomViewsControls.tsx

```diff
@@ -13,17 +13,38 @@ import {
 	openModal,
 	openToast,
 } from 'frontend-js-components-web';
-import {fetch, objectToFormData} from 'frontend-js-web';
-import React, {useContext, useRef, useState} from 'react';
+import {fetch} from 'frontend-js-web';
+import React, {Ref, useContext, useRef, useState} from 'react';
 
 import FrontendDataSetContext from '../../FrontendDataSetContext';
-import ViewsContext from '../../views/ViewsContext';
+import {DEFAULT_FETCH_HEADERS} from '../../constants';
+import getDataSetResourceURL from '../../utils/getDataSetResourceURL';
+import getRandomId from '../../utils/getRandomId';
+import ViewsContext, {ICustomView} from '../../views/ViewsContext';
 import {EViewsActionTypes} from '../../views/viewsReducer';
 
 const DEFAULT_VIEW_ID = 'DEFAULT_VIEW';
 
+const RequiredMark = () => (
+	<>
+		<span className="inline-item-after reference-mark text-warning">
+			<ClayIcon symbol="asterisk" />
+		</span>
+		<span className="hide-accessible sr-only">
```

Comment:
Missing empty line between sibling elements (VSC is complaining about this for me)